system-tests/lib/system-tests.ts

```diff
@@ -22,6 +22,7 @@ require('mocha-banner').register()
 const chalk = require('chalk').default
 const _ = require('lodash')
 let cp = require('child_process')
+const fsExtra = require('fs-extra')
```

Comment:
Tiny nit but it looks like in general in the code base we tend to `const fs = require('fs-extra')` since it's a drop-in replacement for the standard `fs`. 

Just a naming thing but good to keep this consistent.

Reply:
I'll update.
```suggestion
const fs = require('fs-extra')
```